rhp/v4/rhp.go

```diff
@@ -458,6 +465,26 @@ type (
 		Balance types.Currency `json:"balance"`
 	}
 
+	// RPCReplenishAccountsRequest is the request type for RPCReplenishAccounts.
+	RPCReplenishAccountsRequest struct {
+		Accounts           []Account            `json:"accounts"`
+		Target             types.Currency       `json:"target"`
+		ContractID         types.FileContractID `json:"contractID"`
+		ChallengeSignature types.Signature      `json:"challengeSignature"`
+	}
+	// RPCReplenishAccountsResponse is the response type for RPCReplenishAccounts.
+	RPCReplenishAccountsResponse struct {
+		Cost types.Currency `json:"cost"`
```

Comment:
Thinking about it a bit more, do we want to return more info here? This is definitely the most efficient way to design the protocol, but there might be information we need that we can't get from it right now. e.g. how much we actually fund a specific account with. On the other hand it's also a features to not have detailed insights into the spending habits of users.

Reply:
If we decide to return the specific account, we can also change it to fund as many accounts as possible instead of erroring.